templates/ts-crawlee-cheerio/src/main.ts

```diff
@@ -7,19 +7,27 @@
 // For more information, see https://docs.apify.com/sdk/js
 import { Actor } from 'apify';
 // For more information, see https://crawlee.dev
-import { CheerioCrawler } from 'crawlee';
-import { router } from './routes.js';
+import { CheerioCrawler, Dataset, Dictionary } from 'crawlee';
 
 // Initialize the Apify SDK
 await Actor.init();
 
-const startUrls = ['https://apify.com'];
+const { startUrls, maxRequestsPerCrawl } = (await Actor.getInput() as Dictionary) || {};
```

Comment:
you should have an interface here, and pass it to the generic type arg instead of using type assertions

we should use best practices in the templates, this is rather workaround